File: excel/perl/update/excel_update.pl
#! /usr/bin/perl
#
#	excel_update.pl
#
#					Apr/15/2011
#
# ------------------------------------------------------------------
use	strict;
use	warnings;
use	utf8;
use	Encode;
#
#use	Spreadsheet::ParseExcel;
#
use lib '/var/www/data_base/common/perl_common';
use text_manipulate;
use excel_manipulate;
# ------------------------------------------------------------------
print	(encode ('utf-8',"*** 開始 ***\n"));
my $file_excel = $ARGV[0];
my $id_in = $ARGV[1];
my $population_in = $ARGV[2];
print	$id_in . "\t";
print	$population_in . "\n";
#
my %dict_aa = excel_manipulate::excel_read_proc ($file_excel);
%dict_aa = text_manipulate::dict_update_proc ($id_in,$population_in,%dict_aa);
#
excel_manipulate::excel_write_proc ($file_excel,%dict_aa);
#
text_manipulate::dict_display_proc (%dict_aa);
#
print	(encode ('utf-8',"*** 終了 ***\n"));
#
# ------------------------------------------------------------------

File: excel/ruby/update/excel_update.rb
#! /usr/bin/ruby
# -*- coding: utf-8 -*-
#
#	excel_update.rb
#
#					Jul/15/2013
#
# -------------------------------------------------------------------
load '/var/www/data_base/common/ruby_common/text_manipulate.rb'
load '/var/www/data_base/common/ruby_common/excel_manipulate.rb'

# -------------------------------------------------------------------
require 'spreadsheet'

puts	"*** 開始 ***"

file_excel = ARGV[0]
id_in = ARGV[1]
population_in = ARGV[2].to_i
#
puts id_in,population_in


dict_aa = excel_read_proc(file_excel)
dict_aa = dict_update_proc(dict_aa,id_in,population_in)

excel_write_proc(file_excel,dict_aa)



dict_display_proc(dict_aa)

puts	"*** 終了 ***"
#
